package.json

```diff
@@ -26,7 +27,7 @@
     "ts-jest": "^25.2.1",
     "ts-node": "^8",
     "typescript": "^3.8.2",
-    "vtex": "^2.105.1"
+    "vtex": "^2.112.0-beta.11"
```

Comment:
beta?

Reply:
I was just testing, when the PR is approved, I remove this beta deploy and deploy a stable verison